chart/flux/templates/deployment.yaml

```diff
@@ -51,6 +55,9 @@ spec:
             readOnly: true
           - name: git-keygen
             mountPath: /var/fluxd/keygen
+          - name: docker-creds
+            mountPath: /docker-creds
```

Comment:
Can you make this `/etc/fluxd/docker`, just for consistency's sake? (OK, there is not a lot of consistency on display otherwise, but we have to start somewhere ..)

Reply:
Sure 👍